### ISSUE TITLE
Host user is not able to access cert files and directories anymore

### ISSUE THREAD

**Issue by hvorragend** [JrCs/docker-letsencrypt-nginx-proxy-companion#461]
Since a few days my host user (not root) is not able to access the following files anymore:

```
/opt/<dockerstack>/data/nginx/certs/<virtualhost>
/opt/<dockerstack>/data/nginx/certs/accounts
```
It seems to be related with the following commit:
https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion/commit/212e2513

I am using a "normal" user to back up my whole docker environment. Now this is not possible anymore.

Isn't it possible to assign a UID/GID to your docker container, so that we can choose the owner of this file?

**Comment by buchdag** [comment]
Please fall back to tagged version `v1.9.1`. It contains everything but the permissions change.

**Comment by desimaniac** [comment]
It seems that `FILES_UID`, `FILES_GID`, `FILES_PERMS`, and `FOLDERS_PERMS` environmental variables have no effect. I suppose these options aren't ready yet. 

Ref:
https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion/wiki/Persistent-data#ownership--permissions-of-private-keys-and-acme-account-keys

**Comment by buchdag** [comment]
As noted in the wiki that's still worked on and should be merged soon.

**Comment by buchdag** [comment]
I'll try to have the PR submitted this week end.

**Comment by buchdag** [comment]
Fixed by #463, without additional configuration the container should revert files and folders permissions as they were before #436.
